ui/src/app/features/face-recognition/recognition-result/recognition-result.component.ts

```diff
@@ -42,8 +43,13 @@ export class RecognitionResultComponent implements OnDestroy {
 
   canvasSize: ImageSize = { width: 500, height: null };
   faceDescriptionHeight = 25;
+  formattedResult: string;
   private printSubscription: Subscription;
 
+  ngOnInit() {
+    this.formattedResult = resultRecognitionFormatter(this.requestInfo.response);
```

Comment:
this logic should be in ngOnChanges? otherwise this.formattedResult will be only initialised once

Reply:
Yes that right, could you please explain when the other initialize it can be?

Reply:
when you make a request multiple times formattedResult will show only first one

Reply:
Do you mean a request to upload an image, and view the result?

Reply:
Fixed